pkg/api/series.go

```diff
@@ -71,9 +71,9 @@ func series(queryable *query.Queryable) http.HandlerFunc {
 		var sets []storage.SeriesSet
 		var warnings storage.Warnings
 		for _, mset := range matcherSets {
-			s, _, wrn, err := q.Select(false, nil, nil, mset...)
-			warnings = append(warnings, wrn...)
-			if err != nil {
+			s, _ := q.Select(false, nil, nil, mset...)
+			warnings = append(warnings, s.Warnings()...)
+			if s.Err() != nil {
 				respondError(w, http.StatusUnprocessableEntity, err, "execution")
```

Comment:
Should this be changed to `s.Err()` instead of `err`?

Reply:
Missed that! Addressed now.